### ISSUE TITLE
Files from SymbolsExclusionFile.txt must also be excluded from symbol validation 

### ISSUE THREAD

**Issue by epananth** [dotnet/arcade#7523]
- [ ] This issue is blocking 
- [ ] This issue is causing unreasonable pain

After this change is merged -> https://github.com/dotnet/arcade/issues/6804
We need to exclude the files which we are not publishing in symbol validations as well 

cc: @adiaaida

**Comment by markwilkie** [comment]
@epananth - is this part of the release pipeline work?  If so, feel free to add it to your epic.

**Comment by adiaaida** [comment]
Yes, this should be part of the release pipeline work. Adding to our epic.

**Comment by epananth** [comment]
thanks @adiaaida
